bundles/org.openhab.binding.gree/src/main/resources/ESH-INF/i18n/gree_de.properties

```diff
@@ -0,0 +1,65 @@
+# GREE Binding
```

Comment:
You should use UTF-8 encoding for this file (currently ISO-8859-1).

Reply:
hmm, just editing in Eclipse

Reply:
You can change the encoding in the file properties within Eclipse. Better convert the file to UTF-8 beforehand. E.g. with Notepad++.

Reply:
fixed